jsdoc/processors/extract-tags.js

```diff
@@ -195,10 +195,16 @@ module.exports = function extractTagsProcessor(log, parseTagsProcessor, createDo
       }
 
       message += 'Line: ' + badTag.startingLine + ': @' + badTag.tagName + ' ' + description + '\n';
-      badTag.errors.forEach(error => message += '    * ' + error + '\n');
+
+      // An object of type `Tag` in the `badTags` list does not necessarily have an `errors`
+      // field attached. This can happen when the tag does not have a tag definition at all.
+      // We only append more specific error messages if such concrete errors are available.
+      if (badTag.errors !== undefined) {
```

Comment:
Note: I have considered cleaning this logic up even more, by adding `errors` as a default property of the `Tag` prototype, but that could break things in other places, or in AIO etc. So this is the non-breaking way, matching up with what happened before the switch from `_.forEach(errors)` to `errors.forEach(`